crdt.go

```diff
@@ -598,6 +598,15 @@ func (store *Datastore) Delete(key ds.Key) error {
 	return store.publish(delta)
 }
 
+// Sync is a no-op.
+// Before broadcasting deltas we run processNode making sure that every
+// operation (including Put and Delete) applied to this store take effect
+// (delta is merged) before returning.
+func (store *Datastore) Sync(key ds.Key) error {
+	store.logger.Warning("sync is a no-op")
```

Comment:
If sync doesn't do anything, there's no need to log. We should just return nil and move on.